test/cql-pytest/test_cast.py

```diff
@@ -61,11 +61,10 @@ def test_cast_int_literal_to_blob_with_blob_cast(cql, table1):
         cql.execute(f"INSERT INTO {table1} (pk, blob_col) VALUES ({pk}, (blob)123)")
 
 # Putting (int) before the integer specifies the exact type and it's possible to cast the value.
-# This type of assignment isn't currently allowed, will work in the future.
-def test_cast_int_literal_with_type_hint_to_blob(cql, table1):
+def test_cast_int_literal_with_type_hint_to_blob(cql, table1, scylla_only):
```

Comment:
It would be nice to have a comment why the test is scylla_only. Since most of the new scylla_only tests have the same explanation on why they are scylla_only, I think it's enough to have this explanation just once in this file.